example/node/index.js

```diff
@@ -40,43 +61,62 @@ async function run() {
   printOut(`Layer title:`, layer.title);
   printOut('Layer description:', layer.description);
 
-  if (!process.env.AUTH_TOKEN) {
-    printOut('Other examples need an auth token. Set env var AUTH_TOKEN and run again.', null);
+  if (!process.env.CLIENT_ID && !process.env.CLIENT_SECRET) {
```

Comment:
Minor: it should probably be `||` instead of `&&`?

Reply:
Thanks, true. Changed.